### PR TITLE
Testing api for superheroes

### PR DESCRIPTION
**Pull request by giantqtipz**
1) Installed new package called 'supertest' to allow express/routes testing.
2) Created 'endpoint.spec.js' file to test routes.
3) Updated package.json with two scripts 'test' and 'test:endpoint'. Run 'npm run test:endpoint' to test routes.
4) Created two simple tests for Superheroes:
    - Fetch data from /api/superheroes
    - Post data into /api/superheroes

Will create tests for categories and acts too.

**the test:endpoint script uses --forceExit argument because jest doesn't stop running after the test is done. Still looking for a solution so we don't need to --forceExit.

### CODE REVIEW

File: tests/endpoint.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+describe('Superhero Tests', () => {
+  describe('Fetch superheroes', () => {
+    it('Returns status 200', async () => {
+      const res = await request.get('/api/superheroes');
+      expect(res.statusCode).toBe(200);
+    });
+  });
+  describe('Create a new superhero', () => {
+    it('Returns status 200, and the posted objects', (done) => {
+      return request
```
Comment (DmytroLypovetskiy) [inline tests/endpoint.spec.js:15]: posted objects or object?

File: tests/endpoint.spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    });
+  });
+});
+
+describe('Acts Tests', () => {
+  describe('Fetch acts', () => {
+    it('Returns status 200', async () => {
+      const res = await request.get('/api/acts');
+      expect(res.statusCode).toBe(200);
+    });
```
Comment (DmytroLypovetskiy) [inline tests/endpoint.spec.js:41]: what's the difference between using `statusCode ` and just `status`?

File: tests/endpoint.spec.js (hunk 1 above)
Comment (giantqtipz) [reply tests/endpoint.spec.js:15]: object 👍

File: tests/endpoint.spec.js (hunk 2 above)
Comment (giantqtipz) [reply tests/endpoint.spec.js:41]: there is actually no difference. I changed it to res.status in next commit

**Pull request merged**